cfgov/unprocessed/js/modules/UStreamPlayer.js

```diff
@@ -1,14 +1,12 @@
-'use strict';
+const VideoPlayer = require( './VideoPlayer' );
```

Comment:
Outside this scope, but this shouldn't be used any more. Can you add a note to remove this file?